tex/rep-theory/rep-alg.tex

```diff
@@ -330,12 +330,12 @@ \section{Irreducible and indecomposable representations}
 	$a \cdot w \in W$.
 	In other words, this subspace is invariant under actions by $A$.
 \end{definition}
-Thus if $V \neq W_1 \oplus W_2$ for representations $W_1$, $W_2$
-then $W_1$ and $W_2$ are subrepresentations.
+Thus if $V = W_1 \oplus W_2$ for representations $W_1$, $W_2$
+then $W_1$ and $W_2$ are subrepresentations of $V$.
 
 \begin{definition}
 	If $V$ has no proper subrepresentations then it is \vocab{irreducible}.
-	If $V \neq W_1 \oplus W_2$ for proper subrepresentations $W_1$, $W_2$,
+	If $V \neq W_1 \oplus W_2$ for any pair of proper subrepresentations $W_1$, $W_2$ of $V$,
```

Comment:
In the irreducible part, "nonzero" condition should be considered, depending on the definition of subrepresentation above.

In the indecomposable part, how about stating instead: `If there is no pair of proper subrepresentations $W_1$, $W_2$ such that $V = W_1 \oplus W_2$,`

Reply:
Yes, nonzero. I'll take your suggestion for the indecomposable part too.